spec/config_spec.rb

```diff
@@ -70,9 +70,27 @@
     end
     
     it 'can be set to true' do
-      config = Money::Config.new
-      config.experimental_crypto_currencies = true
-      expect(config.experimental_crypto_currencies).to eq(true)
+      configure(experimental_crypto_currencies: true) do
+        expect(Money.config.experimental_crypto_currencies).to eq(true)
+      end
+    end
+
+    it 'can be set to true using the bang method' do
+      configure(experimental_crypto_currencies: true) do
+        expect(Money.config.experimental_crypto_currencies).to eq(true)
+      end
+    end
+  end
+
+  describe 'legacy_json_format' do
+    it 'defaults to false' do
+      expect(Money::Config.new.legacy_json_format).to eq(false)
+    end
+
+    it 'can be set to true using the bang method' do
+      configure(legacy_json_format: true) do
+        expect(Money.config.legacy_json_format).to eq(true)
+      end
```

Comment:
suggestion: Similar to the above. I think if we're testing the bang method we should call that method explicitly within the test, rather than relying on the helper whose implementation might change in the future.